src/components/app-text-field/AppTextField.tsx

```diff
@@ -25,7 +25,9 @@ const AppTextField: FC<AppTextFieldProps> = ({
           {errorMsg}
         </Typography>
       </Tooltip>
-    ) : null
+    ) : (
+      ' '
+    )
```

Comment:
This solution adds whitespace to the bottom of the input. This will fix the issue on the screenshots but it can cause additional whitespace and visual discrepancy to the design in other places